x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -134,11 +144,12 @@ export const defineGetBenchmarksRoute = (router: CspRouter) =>
             cspContext.soClient,
             agentPolicies,
             agentStatusesByAgentPolicyId,
-            cspPackagePolicies.items
+            cspPackagePolicies
           );
 
-          const getBenchmarkResponse: GetBenchmarkResponse = {
-            ...cspPackagePolicies,
+          const getBenchmarkResponse = {
+            ...packagePolicies,
+            total: cspPackagePolicies.length,
```

Comment:
it looks like the change might break the pagination when there is more than one page as you are updating the `total` to match the additional filtering but `page` stays the same. To avoid having a pagination logic here in the route, what about moving the additional filtering you implemented inside  `getCspPackagePolicies`?

Reply:
@Omolola-Akinleye I see that the PR is open for quite a while. Anything I can help with here?

Reply:
Hi  @maxcold  we postponed this PR to quick wins since the solution wasn't as straightforward. I updated the PR now which should address the pagination issue.

https://github.com/elastic/kibana/assets/17135495/4064bb3b-361b-4aeb-8630-d123858e59a3